Project.toml

```diff
@@ -3,13 +3,10 @@ uuid = "276daf66-3868-5448-9aa4-cd146d93841b"
 version = "0.8.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+OpenSpecFun_jll = "efe28fd5-8261-553b-a9e1-b2916fc3738e"
 
 [compat]
-BinaryProvider = "≥ 0.5.0"
-julia = "≥ 0.7.0"
+julia = "1.3"
```

Comment:
1.3 hasn't even been released yet. I don't think we should drop support for 1.0 and later.

Reply:
How do you propose to use the new infrastructure? 
People can still use SpecialFunctions  on < Julia 1.3 .  This was partly my question https://github.com/JuliaMath/SpecialFunctions.jl/pull/187#issuecomment-539586063

Reply:
We can also branch and backport bugfixes to  versions compatible with < Julia 1.3?

Reply:
> How do you propose to use the new infrastructure?

Don't bother yet, at least until 1.3 is released and we're confident that there won't be necessary bugfixes or anything that apply to previous versions of Julia.

Reply:
Sounds good.

Reply:
Anyway I think it's good to have these early pull requests to test the new framework and maybe let brave users try it if they want 🙂

Reply:
Yeah, don't really expected this to get merged - But I needed it anyways as a fork ;)